### ISSUE TITLE
Stats: align post stats to the side

### ISSUE THREAD

**Issue by keoshi** [Automattic/wp-calypso#73159]
Part of the modernization project (p1HpG7-igQ-p2).

#### Problem

In the detailed view of a specific post or page, the metrics that appear at the bottom seem very spaced out.

![Image](https://user-images.githubusercontent.com/390760/217845546-68fd0cc7-ae9e-466f-85d8-366fa6986c19.png)

This becomes even more evident when there's no featured image and the counts stretch across the entire block, making it very difficult to make sense of the layout:

![Image](https://user-images.githubusercontent.com/390760/217845382-a4aeef6f-c7bd-4162-9b37-4b2adc5d8cc8.png)

#### Proposal

We could align the metrics to the side, making the entire structure more compact and creating a visual pattern where similar elements are grouped together.

Took a stab at playing around with some styles and I quite like the results using the following code:

```diff
.post-stats-card__counts {
-    justify-content: space-between;
+    gap: 48px; // or even 64px
}
```

Before | After
--|---
![Image](https://user-images.githubusercontent.com/390760/217844029-862dc961-e025-430f-a457-3ba7cd95345b.png) | ![Image](https://user-images.githubusercontent.com/390760/217843738-dcf90a4e-ecef-4ed9-abdb-2388f99c481e.png)

What do y'all think about this tiny enhancement? cc @jeffgolenski

**Comment by dognose24** [comment]
Fixed in [68cb03f](https://github.com/Automattic/wp-calypso/pull/73144/commits/68cb03fa21957d40783e60ac4f611079bd304ae6) inside the PR https://github.com/Automattic/wp-calypso/pull/73144.

**Comment by jeffgolenski** [comment]
@keoshi I like the spacing! In another PR review I also suggested a max content width of 550px. It also helps when the cards go full width.

**Comment by keoshi** [comment]
@jeffgolenski Yesss! As you know, @dognose24 is absolute boss and already made both of those changes in this PR: https://github.com/Automattic/wp-calypso/pull/73144

**Comment by jsnmoon** [comment]
I believe this is already done? 🙂

**Comment by keoshi** [comment]
Yup, this is done as per @dognose24 above:

> Fixed in [68cb03f](https://github.com/Automattic/wp-calypso/pull/73144/commits/68cb03fa21957d40783e60ac4f611079bd304ae6) inside the PR https://github.com/Automattic/wp-calypso/pull/73144.

Thank you all!
